### ISSUE TITLE
Title is misleading

### ISSUE THREAD

**Issue by ShaunInIdaho** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#643]
This article doesn't discuss adding a bot to a channel, only a private chat even though the title suggests otherwise.  The process for side loading a bot to a channel is much different.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c1257b9b-73ca-8a97-3f65-a7c8d5937e8c
* Version Independent ID: 5f9b5fad-dceb-3f9e-192c-a31ca05e0b83
* Content: [Add bots for private chats and channels in Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/add-bots)
* Content Source: [Teams/add-bots.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/add-bots.md)
* Service: **msteams**
* GitHub Login: @LolaJacobsen
* Microsoft Alias: **lolaj**

**Comment by kenwith** [comment]
@ShaunInIdaho - Thank you for submitting this feedback.

It is a very good catch regarding side loading a bot and how things can be made clearer.

I was able to get this into the backlog for the Teams/Skype writing team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
